client/app/scripts/components/nodes.js

```diff
@@ -73,7 +73,15 @@ class Nodes extends React.Component {
   }
 
   setDimensions() {
-    this.props.setViewportDimensions(window.innerWidth, window.innerHeight);
+    // Use the `scope-app` div to determine the viewport rectangle, because `window` gives
+    // wrong values for scope container inside Weave Cloud (on dev & prod), because of the
+    // top navigation toolbar that is counted in, even though it's not part of Scope.
+    const viewport = document.getElementsByClassName('scope-app')[0].getBoundingClientRect();
+    // Not sure why we need to subtract `left` and `top`, but that gives us correct values.
```

Comment:
It seems to be because of the height: 100% in _base.scss. I think we can remove that?

```

.scope-app, .terminal-app {
  -webkit-font-smoothing: antialiased;
  background: $body-background-color;
  bottom: 0;
  color: $text-color;
  font-family: $base-font;
  font-size: 13px;
  height: 100%;
  --- snip ---
```